src/Compilers/Extension/project.json

```diff
@@ -1,8 +1,12 @@
 ﻿{
   "dependencies": {
     "EnvDTE": "8.0.0",
-    "Microsoft.VisualStudio.Shell.14.0": "14.3.25407",
-    "Microsoft.VisualStudio.Shell.Interop.10.0": "10.0.30319"
+    "Microsoft.VisualStudio.Shell.15.0": "15.0.26014-alpha",
```

Comment:
I was indicating that we should move to the official RC3 packages where applicable and have the custom packages come from the same build, for maximum compatibility